### DISCUSSION TITLE
Docs: Copy to Clipboard button

### DISCUSSION THREAD

**Discussion by adsingh14** [vercel/next.js#42081] [Ideas]
### What is the improvement or update you wish to see?

For Copy to clipboard button in code block. Check the yellow box in attachment [^1]

[^1]: 
![image](https://user-images.githubusercontent.com/35252877/197853991-e668141a-f218-4210-96d0-b83bfbe9248c.png)

### Is there any context that might help us understand?

```- Same as the above -```

### Does the docs page already exist? Please link to it.

https://beta.nextjs.org/docs/getting-started

**Comment by kumarajay0412** [comment]
It would be great option to add a copy to clipboard button in every block of code and i think most website use this and from point of good UX also this is a right decision

**Comment by adsingh14** [comment]


**Reply by kumarajay0412** [reply]
I am also willing to contribute to this issue

**Comment by FrozenHearth** [comment]
Was just gonna raise this issue. This is very much needed.
